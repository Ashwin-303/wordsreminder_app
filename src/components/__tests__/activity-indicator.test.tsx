import React from 'react';
import renderer from 'react-test-renderer';
import { ActivityIndicator } from '../activity-indicator';

describe('ActivityIndicator', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<ActivityIndicator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
