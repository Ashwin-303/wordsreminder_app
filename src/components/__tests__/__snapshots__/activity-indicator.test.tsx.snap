// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityIndicator should match snapshot 1`] = `
<ActivityIndicator
  accessibilityHint="Loading"
  animating={true}
  color="#ffffff"
  hidesWhenStopped={true}
  size="small"
  style={
    Object {
      "padding": 10,
    }
  }
/>
`;
